alf/algorithms/off_policy_algorithm.py

```diff
@@ -229,6 +229,39 @@ def _update(self, experience, weight):
 
         info = alf.nest.utils.stack_nests(info_list)
         info = dist_utils.params_to_distributions(info, self.train_info_spec)
+        return info
+
+    def _collect_train_info_parallelly(self, experience):
+        batch_size = experience.step_type.shape[1]
+        length = experience.step_type.shape[0]
+
+        exp = alf.nest.map_structure(lambda x: x.reshape(-1, *x.shape[2:]),
+                                     experience)
+
+        if self._use_rollout_state:
```

Comment:
Based on the code here,  I think we should add comments clarifying that `temporally_independent_train_step` can also be set to True if we don't recompute the state but just take the historical states from the replay buffer.

Reply:
Noted in TrainerConfig